frontend/src/components/SideNav.tsx

```diff
@@ -277,6 +278,18 @@ export default class SideNav extends React.Component<SideNavProps, SideNavState>
               </Button>
```

Comment:
I don't think this necessarily needs to block this PR, but we should keep the `Archive` tab highlighted in the sidenav even if a user starts digging into a run/resource. Currently, once they click into a run, the sidenav goes back to highlighting `Experiments`

Reply:
Interesting point. I think I agree, but the current set up doesn't lend itself easily to do this, we can think about how to do this.
What we can do too is change the breadcrumbs to include the archive instead of the run's regular path, since you wouldn't be able to see the run anyway if you go to its experiment.